<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1255</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Intelligent Scissor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>511</width>
      <height>451</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>511</width>
      <height>451</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>701</width>
      <height>551</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1255</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSave">
     <property name="title">
      <string>Save</string>
     </property>
     <addaction name="actionSave_Contour"/>
     <addaction name="actionSave_Mask"/>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="menuSave"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionCost_Graph"/>
    <addaction name="actionPath_Tree"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionWork_Mode"/>
    <addaction name="actionDebug_Mode"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuDebug"/>
   <addaction name="menuMode"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionWork_Mode"/>
   <addaction name="actionDebug_Mode"/>
   <addaction name="actionSave_Contour"/>
   <addaction name="actionSave_Mask"/>
   <addaction name="separator"/>
   <addaction name="actionCost_Graph"/>
   <addaction name="actionPath_Tree"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_Contour">
   <property name="text">
    <string>Save Contour</string>
   </property>
  </action>
  <action name="actionSave_Mask">
   <property name="text">
    <string>Save Mask</string>
   </property>
  </action>
  <action name="actionCost_Graph">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cost Graph</string>
   </property>
  </action>
  <action name="actionWork_Mode">
   <property name="text">
    <string>Work_Mode</string>
   </property>
  </action>
  <action name="actionDebug_Mode">
   <property name="text">
    <string>Debug_Mode</string>
   </property>
  </action>
  <action name="actionPath_Tree">
   <property name="text">
    <string>Path Tree</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
